js/addToTask.js

```diff
@@ -0,0 +1,68 @@
+import {
+  CREATE_TASK_BUTTON,
+  CONTENT,
+  UL_LIST,
+  INPUT_TASK,
+  INPUT,
+  ADD_TASK,
+  TRASH_INPUT,
+  TODAY,
+  COMPLETE,
+  EDIT,
+  DELETE_TODO,
+  CLICK_EVENT,
+  HIDE,
+  AFTER_BEGIN,
+} from "/js/constants.js"; //named import
+
+export let taskList = [];
+export let id = 0;
```

Comment:
Why are we exporting these? Are we using them elsewhere? if so, let's make an object out of these and export that object instead, which organizes the code better.

Reply:
taskList should be const. I have updated it to const and id to let. no need to export these.